integration-cli/docker_cli_run_test.go

```diff
@@ -146,34 +146,6 @@ func (s *DockerSuite) TestRunWorkingDirectory(c *check.C) {
 	}
 }
 
-// pinging Google's DNS resolver should fail when we disable the networking
-func (s *DockerSuite) TestRunWithoutNetworking(c *check.C) {
```

Comment:
-n and --networking are deprecated, nothing to do with --net

The first part of the test is testing --net so it should be kept, the other -n -- I'll remove it